source/developer/bot-accounts.rst

```diff
@@ -33,36 +33,37 @@ If you would like to see improvements to bot accounts, `let us know in the Featu
   :local:
 
 Configuration Settings
-------------------------
+----------------------
 
 By default, plugins can create and manage bot accounts. To enable bot account creation through the user interface or the RESTful API:
 
 1. Go to **System Console > Integrations > Bot Accounts**.
 2. Set **Enable Bot Account Creation** to ``true``.
 
-Once set, System Administrators can create bot accounts for integrations using the **Integrations > Bot Accounts** link in the description provided.
+Once set, System Admin can create bot accounts for integrations using the **Integrations > Bot Accounts** link in the description provided.
```

Comment:
What's our standard for "System Administrators" versus "System Admins" in product documentation?

A suggestion to consider: Write out "System Administrator/s" fully for the first reference on a given page, and then use "System Admin/s" thereafter. Thoughts?

Reply:
We refer to System Admins relatively consistently throughout the docs (or more consistently than not). There are instances that still need to be changed, but the standard is "System Admin" which I realize is at odds with the UI which uses "Administrator" (but I believe that's the only glaring outlier).

Reply:
Thanks, @justinegeffen!